editor/src/clj/editor/breakpoints_view.clj

```diff
@@ -0,0 +1,526 @@
+        :cell-factory {:fx/cell-type fx.table-cell/lifecycle
+                       :describe (fn/partial column-enabled-cell-factory project swap-state breakpoints)}}
+       {:fx/type fx.table-column/lifecycle
+        :text (localization-state (localization/message "breakpoints.column.line"))
+        :pref-width 50
+        :min-width 50
+        :max-width 100
+        :cell-value-factory identity
+        :cell-factory {:fx/cell-type fx.table-cell/lifecycle
+                       :describe (fn/partial column-line-cell-factory breakpoints)}}
+       {:fx/type fx.table-column/lifecycle
+        :text (localization-state (localization/message "breakpoints.column.name"))
+        :pref-width 200
+        :cell-value-factory identity
+        :cell-factory {:fx/cell-type fx.table-cell/lifecycle
+                       :describe (fn/partial column-name-cell-factory breakpoints)}}
+       {:fx/type fx.table-column/lifecycle
+        :text (localization-state (localization/message "breakpoints.column.condition"))
+        :pref-width 250
+        :cell-value-factory identity
+        :cell-factory {:fx/cell-type fx.table-cell/lifecycle
+                       :style-class ["condition-cell"]
+                       :describe (fn/partial column-condition-cell-factory state swap-state project breakpoints)}}
+       {:fx/type fx.table-column/lifecycle
+        :text (localization-state (localization/message "breakpoints.column.path"))
+        :style-class ["path-cell"]
+        :pref-width 200
+        :cell-value-factory identity
+        :cell-factory {:fx/cell-type fx.table-cell/lifecycle
+                       :describe (fn/partial column-path-cell-factory breakpoints)}}
+       {:fx/type fx.table-column/lifecycle
+        :pref-width 50
+        :reorderable false
+        :resizable false
+        :sortable false
+        :cell-value-factory identity
+        :cell-factory {:fx/cell-type fx.table-cell/lifecycle
+                       :describe (fn/partial column-remove-btn-cell-factory swap-state project breakpoints)}}]}}))
+
+(fxui/defc breakpoints-view
+  {:compose [{:fx/type fx/ext-watcher
+              :ref (:localization (:context props))
+              :key :localization-state}
+             {:fx/type fx/ext-state
+              :initial-state {:breakpoints (:breakpoints (:context props))
+                              :edited-breakpoint nil}}]}
+  [{:keys [context localization-state parent state swap-state]}]
+  (let [project (:project context)
+        open-resource-fn (:open-resource-fn context)]
+    {:fx/type fxui/ext-with-anchor-pane-props
+     :desc {:fx/type fxui/ext-value :value parent}
+     :props {:children [(breakpoints-toolbar-view project localization-state state swap-state)
+                        (breakpoints-table-view project open-resource-fn localization-state state swap-state)]}}))
+
+(g/defnk produce-breakpoints-ui [parent-view open-resource-fn breakpoints workspace project prefs localization]
```

Comment:
The `produce-` prefixed `defnk` should match the output name. The output is named `anchor-pane`.